Math/NumberTheory/Primes/Factorisation/LinearAlgebra.hs

```diff
@@ -0,0 +1,135 @@
+module Math.NumberTheory.Primes.Factorisation.LinearAlgebra
+  ( SBVector(..)
+  , SBMatrix(..)
+  , mult
+  , size
+  , linearSolve
+  , testLinearSolver
+  ) where
+
+import qualified Data.List as L
+import qualified Data.IntSet as S
+import qualified Data.Vector as V
+import qualified Data.Vector.Unboxed as U
+import Debug.Trace
+import Data.Semigroup()
+import System.Random
+import System.IO.Unsafe
+import GHC.Clock
+import Data.Bit
+import Data.Bits
+
+-- Sparse Binary Vector
+newtype SBVector = SBVector
+  { set :: S.IntSet
```

Comment:
An established pattern is to call newtype's fields with a prefix `un`, so `unSBVector` in this case.

Reply:
I deleted the getter all together. I don't use it anywhere